k8s/eks/README.md

```diff
@@ -0,0 +1,284 @@
+This guide will help you setup an EKS (Elastic Kubernetes Service) cluster, and run testground plans on it.
+
+## Requirements
+
+It is assumed that you already have the following:
+
+- A bastion host on AWS, or a laptop with AWS credentials which will be used to run the installation script, interact with the cluster, and run testground plans
```

Comment:
AWS docs are not complete regarding this; I have created a step-by-step instruction which will be shared with you.

Reply:
Can you publish it as a draft so that we can already have some insight?

Reply:
```
1. Create a keypair

AWS Console -> EC2 -> Network & Security -> Key Pairs -> Create key pair (upper right corner)
Input key name
Pick RSA
Pick .pem as you will be creating a linux bastion host
Create the key and download the private key. You will use it to connect to the bastion

2. Create an instance

AWS Console -> EC2 -> Instances -> Launch instances (upper right corner)
Add an instance name
Pick Quick Start and Amazon Linux
Instance type t2.micro
Pick your key pair
Pick a VPC - either an existing one (for example the default VPC), or create a new VPC
Pick Create a new Security Group and allow SSH traffic only from your public IP, not 0.0.0.0/0
Make sure that 'Auto-assign public IP' is set to 'Enable'
Root volume (storage) can be 30GB gp2

3. Once the instance shows as 'Running' with both Status Checks as '2/2 checks passed', you may attempt to ssh into the bastion host
on macOS you may use:

ssh -i key-file ec2-user@IP-Address (obtain the public IP address of the bastion host through the AWS Console)
key-file is the private key file you downloaded in step 1.

4. Update the instance and install necessary software listed in the README

Since using Amazon Linux, it should be:

sudo yum update

And then you may install the rest of the software listed in README. 

Note: Amazon Linux is a RHEL/CentOS derivative

5. After everything is installed, pull the infra and testground repos, and continue following the README
```

Reply:
I think it'd be better with a link to https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/get-set-up-for-amazon-ec2.html

Reply:
Yeah sure, we can add that as well.

Reply:
Added the link to the README.